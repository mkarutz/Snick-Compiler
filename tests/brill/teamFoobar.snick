proc main ()
  int x;
  float y;

  x := 1;
  y := 2.0;
  # test automatic type casting
  write x / y;

  y := 0.0;
  # divide-by-zero runtime error
  write x / y;
end
