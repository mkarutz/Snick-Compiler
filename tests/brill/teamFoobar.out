0.500000inf
